src/App.tsx

```diff
@@ -1,11 +1,31 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components/ErrorNotification';
+import { FooterTodo } from './components/FooterTodo';
+import { HeaderTodo } from './components/HeaderTodo';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { StatusToFilter } from './types/StatusToFilter';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterStatus, setFilterStatus] = useState(StatusToFilter.All);
+  const [isError, setIsError] = useState(false);
+  const [title, setTitle] = useState('');
+  const [errorText, setErrorText] = useState('');
+  const [isAdding, setIsAdding] = useState(false);
+  const [TodoIdsToDelete, setTodoIdsToDelete] = useState<number[]>([0]);
```

Comment:
```suggestion
  const [todoIdsToDelete, setTodoIdsToDelete] = useState<number[]>([0]);
```

Reply:
When I saw it below in code, I thought it is a component because it starts from head letter